Source/SwiftLintBuiltInRules/Rules/RuleConfigurations/LineLengthConfiguration.swift

```diff
@@ -14,6 +14,8 @@ struct LineLengthConfiguration: RuleConfiguration {
     private(set) var ignoresComments = false
     @ConfigurationElement(key: "ignores_interpolated_strings")
     private(set) var ignoresInterpolatedStrings = false
+    @ConfigurationElement(key: "excluded_patterns")
```

Comment:
The name should indicate that the exclusion applies to lines:
```suggestion
    @ConfigurationElement(key: "excluded_lines_patterns")
```